sample-android/build.gradle.kts

```diff
@@ -58,7 +58,7 @@ android {
 
     buildTypes {
         getByName("debug") {
-            setMatchingFallbacks("release")
+            setMatchingFallbacks("debug", "release")
```

Comment:
Does that mean we can get rid of `Debug` in the releases?

Reply:
we shouldn't need debug for `release`, but for `debug` we might need `release` as fallback as not all dependencies have been published with the right dependencies